profile-widget/divecartesianaxis.cpp

```diff
@@ -130,11 +131,24 @@ static double sensibleInterval(double inc, int decimals, bool is_time_axis)
 
 	double digits_factor = pow(10.0, digits);
 	int inc_int = std::max((int)ceil(inc / digits_factor), 1);
-	// Do "nice" increments of the leading digit. In general: 1, 2, 4, 5.
-	if (inc_int > 5)
-		inc_int = 10;
-	if (inc_int == 3)
-		inc_int = 4;
+	if (is_metric_depth)
+	{
+		// Do increments quantized to 3. In general: 1, 3, 6, 15
+		if (inc_int > 6)
+			inc_int = 15;
+		else if (inc_int > 3)
+			inc_int = 6;
+		else if (inc_int == 2)
+			inc_int = 3;
+	}
+	else
+	{
```

Comment:
can you take a look at the coding style document 
https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md

this should be
```
        } else {
```